main.go

```diff
@@ -29,7 +29,7 @@ import (
 
 var (
 	// Version holds the current Gitea version
-	Version = "1.9.0-dev"
+	Version = "1.12.0-dev"
```

Comment:
```suggestion
	Version = "develop"
```
as vote suggested ...